### DISCUSSION TITLE
Can I use Frontmatter CMS in a monorepo.

### DISCUSSION THREAD

**Discussion by louiss0** [estruyf/vscode-front-matter#533] [Q&A] [answered]
I don't know If I can use Frontmatter CMS in a mono repo setup. I want to do this because I want to put lots of the projects that I create in one place. I'm currently using Astro for most of my sites.

**Comment by estruyf** [comment, marked as answer]
@louiss0 do you have a project you can share to see if we can make it work? I have never tried it, but I cannot yet think why it wouldn't be possible.

At the root, you can have the `frontmatter.json` file, and you can configure your content folders and static/public folder for your images.

**Reply by louiss0** [reply]
I think you should fork this repo. I created it quickly. If you need help just email me https://github.com/louiss0/frontmatter-cms-test. You can use this repo to conduct experiments I promise I will not touch it unless you tell me to. 

This repo is just an nx mono repo. Remember to just focus on the apps folder and ignore all other folder in this repo.

**Reply by estruyf** [reply]
Hi @louiss0, I just did a PR to the repo with the configuration of Front Matter. The trick is using the explorer panel context menu to link your page folders.

![Xnapper-2023-03-14-09 20 00](https://user-images.githubusercontent.com/2900833/224939100-634dd740-6db6-40e1-bfe5-f79cfe0768dc.png)

**Reply by louiss0** [reply]
Thanks for the help I think you should take it and use it as an example. I'll merge the PR but people need to know how to use this extension in a monorepo set up.
